ClientApp/src/services/projectService.ts

```diff
@@ -184,26 +184,27 @@ export const sendProjectEvaluation = async (
     })?.employee || undefined;
 
   const request: ProjectUpdateRequest = {
-    clientId: project.client?.clientId,
-    endDate: project.endDate,
+    clientId: project.client?.clientId || -1,
+    endDate: project.endDate || "",
     evaluationDeadline: project.evaluationDeadline,
     evaluations:
       project.evaluations?.map(
         (evaluation: EvaluationOverview): EvaluationRequest => {
           return {
-            employee: evaluation.employee,
-            projectRole: evaluation.projectRole,
+            employee: evaluation.employee || fakeNoNameEmployeeSnippet,
+            projectRole: evaluation.projectRole || ProjectRole.CSM,
           };
         },
       ) || [],
-    links: project.links?.filter((link: LinkSnippet): boolean => {
-      return link.url !== undefined;
-    }),
-    managerId: manager?.employeeId,
+    links:
+      project.links?.filter((link: LinkSnippet): boolean => {
+        return link.url !== undefined;
```

Comment:
```suggestion
        return link.url !== undefined && link.url !== "";
```
This should fix the validation error